backend/app/resources/onboarding_request_dto.py

```diff
@@ -1,20 +1,41 @@
+import datetime
+from ..models.onboarding_request import ONBOARDING_REQUEST_STATUSES
+from .validate_utils import validate_userinfo
+
+
 class OnboardingRequestDTO:
     def __init__(
         self,
-        email,
-        organization_address,
-        organization_name,
-        role,
-        primary_contact,
-        onsite_contacts,
+        id,
+        info,
         date_submitted,
         status,
     ):
-        self.email = email
-        self.organization_address = organization_address
-        self.organization_name = organization_name
-        self.role = role
-        self.primary_contact = primary_contact
-        self.onsite_contacts = onsite_contacts
+        self.id = id
+        self.info = info
         self.date_submitted = date_submitted
         self.status = status
+        # TODO: Should we add validation calls to each DTO?
```

Comment:
Yes, at the initialization of each DTO, as the fields are initialized.. we should call validate to ensure they match the requirements we expect.